src/index.html

```diff
@@ -4,9 +4,509 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__top">
+        <a class="header__logo" href="#">
+          <img src="../src/images/logo.png" alt="Logotype">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#">
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Смартфоны и телефоны
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Гаджеты
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Планшеты
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Фото
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">
+                Видео
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="card-container">
+        <section class="card" data-qa="card">
```

Comment:
I see no styles for `main` class. You could delete it. Better name for `card-container` - `catalog`. And you could use mix. 
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

```suggestion
    <main>
      <div class="catalog">
        <section class="card catalog__card" data-qa="card">
```